src/components/microlearning/styles/VideoDetailPage.scss

```diff
@@ -29,6 +29,7 @@
 
   .video-player-container-with-transcript {
     display: flex;
+    padding-bottom: 55px;
```

Comment:
[Qurios] After applying this fixed padding, the video player will remain consistent for all videos.

Reply:
@jajjibhai008 If you're asking whether the video player will stay consistent, yes, it will.